molgenis-security/src/main/java/org/molgenis/security/permission/PermissionManagerServiceImpl.java

```diff
@@ -79,14 +93,108 @@ public PermissionManagerServiceImpl(DataService dataService, GrantedAuthoritiesM
 	public Permissions getGroupPluginPermissions(String groupId)
 	{
 		Group group = dataService.findOneById(GROUP, groupId, Group.class);
-		if (group == null) throw new RuntimeException("unknown group id [" + groupId + "]");
+		if (group == null)
+		{
+			throw new RuntimeException("unknown group id [" + groupId + "]");
+		}
+		return getPluginPermissions(group);
+	}
 
-		List<Authority> groupPermissions = getGroupPermissions(group);
-		Permissions permissions = createPermissions(groupPermissions, SecurityUtils.AUTHORITY_PLUGIN_PREFIX);
```

Comment:
The switch in the createPermissions function is now obsolete, it is never called with anything else than AUTHORITY_ENTITY_PREFIX